packages/@okta/vuepress-site/docs/guides/customize-tokens-returned-from-okta/create-app-profile-attribute/index.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Include app-specific information in a custom claim
+---
+In machine to machine flows, if you want to include certain app-specific information in a token claim, you can do so by first adding the metadata into the profile section of the service app. You can access any values that are put inside the app profile using `app.profile` written in [Okta Expression Language](/docs/reference/okta-expression-language/).
```

Comment:
This will work with all flows. So we could start with "If you want to include....."